scripts/check-examples.sh

```diff
@@ -3,7 +3,7 @@
 for folder in examples/* ; do
   cp -n packages/create-next-app/templates/default/gitignore $folder/.gitignore;
   if [ -f "$folder/package.json" ]; then
-    cat $folder/package.json | jq '.license = "MIT"' | sponge $folder/package.json
+    cat $folder/package.json | jq '.license = "MIT" | .private = true' | sponge $folder/package.json
```

Comment:
It doesn't look like we are running this in CI currently, can we add this to the lint job here https://github.com/vercel/next.js/blob/965a280bc2b74a704cecb3098f3be084eba5f15e/.github/workflows/build_test_deploy.yml#L44

Reply:
@ijjk I was actually creating a separate PR for that! (#28009) Just so it'd be a bit clearer what each PR does 😄 Also, the job would fail until the other PRs that does various fixes to examples should probably be merged first so that everything is fine once the CI starts checking it